src/main/java/org/jabref/gui/Dark.css

```diff
@@ -66,6 +66,22 @@
     -fx-background-color: -fx-control-inner-background;
 }
 
+.table-row-cell: odd{
+    -fx-background-color:#272b38 ;
+    -fx-background-insets: 0, 0 0 1 0;
```

Comment:
I think the insets and padding don't need to be set here again, as they are controlled by the light theme, right?

Reply:
Yes, you are right. I'll change it.